runner/local/simple.go

```diff
@@ -195,10 +209,18 @@ func (r *simpleRunner) run(cmd *runner.Command, runId runner.RunId, doneCh chan
 
 	switch st.State {
 	case execer.COMPLETE:
-		r.updateStatus(runner.CompleteStatus(runId, st.ExitCode))
+		r.updateStatus(runner.CompleteStatus(runId, st.ExitCode), r.runnerAvailableCh)
 	case execer.FAILED:
-		r.updateStatus(runner.ErrorStatus(runId, fmt.Errorf("error execing: %v", st.Error)))
+		r.updateStatus(runner.ErrorStatus(runId, fmt.Errorf("error execing: %v", st.Error)), r.runnerAvailableCh)
 	default:
-		r.updateStatus(runner.ErrorStatus(runId, fmt.Errorf("unexpected exec state: %v", st.State)))
+		r.updateStatus(runner.ErrorStatus(runId, fmt.Errorf("unexpected exec state: %v", st.State)), r.runnerAvailableCh)
+	}
+}
+
+func reportRunnerAvailable(runnerAvailableCh chan struct{}) {
```

Comment:
Glad you mentioned this - I forgot a //TODO message:  should reportRunnerAvailable() be called when updateStatus() doesn't find the runId (line113) - if yes, then I would leave reportRunnerAvailable() in place (it hits my 'don't copy >=3 loc rule)